### DISCUSSION TITLE
Kong health check not valid, got http 426

### DISCUSSION THREAD

**Discussion by yulogs** [Kong/kong#8675] [Help] [answered]
I install kong in openshift.
Target set k8s istio ingress gateway, connect route service is ok, but  health check not valid, it show got http 426. I google some solution, i change istio ingress gateway support http 1.0, then ok. 
Why ? Kong call health check using http 1.0? But I see proxy_http_version:1.1? What can I set?
Please help me.....

**Comment by mayocream** [comment, marked as answer]
Hi @yulogs, 

If you mean the health check between Kong and the upstream server, the answer is:

- The active health check for HTTP endpoints of Kong relies on the [lua-resty-healthcheck](https://github.com/Kong/lua-resty-healthcheck) library, the library uses HTTP/1.0 to send requests to upstream.
  https://github.com/Kong/lua-resty-healthcheck/blob/0440b43e6b4595c4e7e5e476da4ae9a135162d12/lib/resty/healthcheck.lua#L997

**Reply by yulogs** [reply]
Thank you....

**Comment by hoangphuocbk** [comment]
I had same issue with Kong health check when HTTP 1.0 protocol is not allowed by other softwares :(
